### PR TITLE
Create a file for 2019 starterkit

### PR DESCRIPTION
**Pull request by vlisovsk**
I have realised that the 2019 starterkit is not featured at the main webpage.
Added a copypaste of the 2018 announcement.

### CODE REVIEW

File: _posts/2019-10-21-starterkit-autumn-2019 [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Registration is now open! You can find all necessary information, including the
+registration form on [the Indico page for the event][indico].
+
+The LHCb Starterkit is a yearly workshop designed to teach new students at LHCb
+the computing skills they need to become effective researchers. This year, we
+will once again teach the fundamentals of the LHCb software, as well as useful
+basics like Bash, Git and Python to 45 Masters and PhD students from LHCb.
+
+Additionally this year there will be 30 students from ALICE and 10 students from SHiP collaborations joining for the
```
Comment (chrisburr) [inline _posts/2019-10-21-starterkit-autumn-2019:22]: @marthaisabelhilton Do you know if there will be any SHiP/ALICE involvement?

Comment (marthaisabelhilton) [reply _posts/2019-10-21-starterkit-autumn-2019:22]: @chrisburr Yes the plan is as usual to have SHiP and ALICE there. I have met with the organisers from both experiments a few weeks ago and they planned to open registration then. I will get in touch with them to see how things are going. We have three groups for the first two days as usual with ALICE contributing some helpers and teachers.

Comment (chrisburr) [reply _posts/2019-10-21-starterkit-autumn-2019:22]: Wonderful! 👍

**Comment by chrisburr** [comment]
Thanks @vlisovsk! 🍰

**Comment by vlisovsk** [comment]
Hmm, it did not appear at the webpage yet. Did I forget to edit something else?

**Comment by chrisburr** [comment]
The `.md` extension was missing, [it's fixed now](https://github.com/lhcb/starterkit/commit/61e88bdb4fd36c2fcd4edc6c4fa2f81694faaae3).

**Pull request merged**